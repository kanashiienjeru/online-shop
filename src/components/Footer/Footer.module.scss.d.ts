declare const styles: {
  readonly "container": string;
  readonly "content": string;
  readonly "title": string;
  readonly "aboutUs": string;
  readonly "input": string;
  readonly "menu": string;
  readonly "categories": string;
  readonly "priceList": string;
  readonly "socials": string;
  readonly "socialsList": string;
  readonly "contacts": string;
  readonly "mail": string;
  readonly "cards": string;
};
export = styles;

