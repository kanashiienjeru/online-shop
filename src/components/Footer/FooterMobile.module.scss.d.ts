declare const styles: {
  readonly "footer": string;
  readonly "aboutUs": string;
  readonly "aboutUsHead": string;
  readonly "input": string;
  readonly "links": string;
  readonly "title": string;
  readonly "contacts": string;
  readonly "contactsHead": string;
  readonly "call": string;
  readonly "socials": string;
  readonly "socialsList": string;
  readonly "mail": string;
  readonly "cards": string;
};
export = styles;

