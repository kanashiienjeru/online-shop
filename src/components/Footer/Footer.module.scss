// @type


footer {
    background: #3F4E65;
    margin-top: 100px;
    display: flex;
}

.container {
    width: 72%;
    margin: 0 auto;

    @media (max-width: 1790px) {
        width: 90%;
    }

    @media (max-width: 1440px) {
        width: 95%;
    }
}

.content {
    margin-top: 71px;
    display: flex;
    justify-content: center;
    gap: 95px;
    font-family: 'Inter';
    font-style: normal;
    color: #FFFFFF;

    @media (max-width: 1350px) {
        gap: 50px;
    }

    @media (max-width: 1000px) {
        display: grid;
        grid-template-areas:
        "aboutUs aboutUs aboutUs contacts contacts contacts"
        "menu menu categories categories priceList priceList";
        margin-bottom: 80px;
    }

    .title {
        font-weight: 600;
        font-size: 20px;
        line-height: 150%;
        margin-bottom: 25px;

        @media (max-width: 1350px) {
            font-size: 17px;
        }

        @media (max-width: 1000px) {
            font-size: 15px;
        }
    }

    p {
        font-weight: 400;
        font-size: 14px;
        line-height: 150%;

        @media (max-width: 1350px) {
            font-size: 11px;
        }
    }

    p:not(:last-of-type) {
        margin-bottom: 20px;

        @media (max-width: 1350px) {
            margin-bottom: 15px;
        }
    }

    .aboutUs {
        margin-right: -35px;
        @media (max-width: 1000px) {
            justify-self: center;
            grid-area: aboutUs;
        }

        & > img {
            margin-bottom: 15px;

            @media (max-width: 1350px) {
                width: 140px;
            }
        }

        p {
            width: 320px;
            font-weight: 400;
            font-size: 16px;
            line-height: 150%;
            margin-bottom: 45px;

            @media (max-width: 1350px) {
                font-size: 13px;
                width: 250px;
            }
        }

        p:last-of-type {
            font-size: 12px;
            margin-bottom: 14px;

            @media (max-width: 1350px) {
                font-size: 11px;
            }
        }

        .input {
            margin-bottom: 116px;
            position: relative;

            @media (max-width: 1350px) {
                margin-bottom: 80px;
            }

            @media(max-width: 1000px) {
                margin-bottom: 10px;
            }
            input {
                width: 180px;
                background-color: #fff;
                padding: 20px;
                padding-right: 60px;
                color: #000;
                border-radius: 36px;

                font-family: 'Inter';
                font-style: normal;
                font-weight: 400;
                font-size: 11px;

                @media (max-width: 1350px) {
                    width: 150px;
                    padding: 10px;
                    padding-right: 40px;
                
                }

                &::placeholder {
                    font-family: 'Inter';
                    font-style: normal;
                    font-weight: 400;
                    font-size: 11px;
                    line-height: 150%;
                    color: rgba(92, 99, 112, 0.6);

                }
            }

            img {
                position: absolute;
                top: 10px;
                right: 68px;
                padding: 10px 15px;
                background: #FFC85E;
                border-radius: 1000px;
                margin: 0;
                cursor: Pointer;

                @media (max-width: 1350px) {
                    right: 57px;
                    top: 6px;
                    padding: 4px 8px;
                }
            }
        }
    }

    .menu {
        @media (max-width: 1000px) {
            grid-area: menu;
        }
    }

    .categories {
        @media (max-width: 1000px) {
            grid-area: categories;
        }
    }

    .menu, .categories {
        p {
            cursor: pointer;
        }
    }
    .priceList {
        @media (max-width: 1000px) {
            grid-area: priceList;
        }

        button {
            width: 192px;
            padding: 21px 0;
            background-color: #FFC85E;
            border-radius: 79px;
            cursor: pointer;

            display: flex;
            align-items: center;
            justify-content: center;
            gap: 13px;

            font-family: 'Inter';
            font-style: normal;
            font-weight: 700;
            font-size: 14px;
            line-height: 17px;
            color: #FFFFFF;

            @media (max-width: 1350px) {
                width: 140px;
                font-size: 12px;
                padding: 10px 0;
            }
        }

        .socials {
            margin-top: 20px;

            .socialsList {
                margin-top: 15px;
                display: flex;
                gap: 10px;
                justify-content: flex-start;
                align-items: center;
                flex-wrap: wrap;

                @media (max-width: 1350px) {
                    margin-top: 10px;
                }

                & > img {
                    @media (max-width: 1350px) {
                        width: 30px
                    }
                }
            }
        }
    }


    .contacts {

        @media (max-width: 1000px) {
            grid-area: contacts;
            align-self: center;
            justify-self: flex-end;
        }

        p:first-of-type {
            font-weight: 600;
            margin-bottom: 5px;
        }

        p:nth-of-type(2) {
            font-weight: 300;
            font-size: 12px;
            line-height: 150%;
        }

        a {
            font-family: 'Inter';
            font-size: 10px;
            text-decoration-line: underline;
            color: #fff;
        }

        .mail {
            margin: 25px 0;

        }

        .cards {
            display: flex;
            align-items: center;
            gap: 5px;


            img {
                @media (max-width: 1350px) {
                    width: 45px
                }
            }
        }
    }

}