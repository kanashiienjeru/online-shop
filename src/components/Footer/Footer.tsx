import styles from "./Footer.module.scss";

const Footer = () => {
  return (
    <div className={styles.container}>
      <div className={styles.content}>
        <div className={styles.aboutUs}>
          <img src="/images/logo/whiteLogo.svg" alt="logo" />
          <p>
            Компания «Султан» — снабжаем розничные магазины товарами "под ключ"
            в Кокчетаве и Акмолинской области
          </p>

          <p>Подпишись на скидки и акции</p>
          <div className={styles.input}>
            <input placeholder="Введите ваш E-mail" type="email" />
              <img src="/images/arrows/arrow.svg" alt="arrow" />
          </div>
        </div>

        <div className={styles.menu}>
            <h4 className={styles.title}>Меню сайта:</h4>
            <p>О компании</p>
            <p>Доставка и оплата</p>
            <p>Возврат</p>
            <p>Контакты</p>
        </div>

        <div className={styles.categories}>
            <h4 className={styles.title}>Категории:</h4>
            <p>Бытовая химия</p>
            <p>Косметика и гигиена</p>
            <p>Товары для дома</p>
            <p>Товары для детей и мам</p>
            <p>Посуда</p>
        </div>

        <div className={styles.priceList}>
            <h4 className={styles.title}>Скачать прайс-лист:</h4>
            <button>
                Прайс-лист
                <img src="/images/icons/download.svg" alt="download" />
            </button>
            <div className={styles.socials}>
            <p>Связь в мессенджерах:</p>
            <div className={styles.socialsList}>
                <img src="/images/footer-pictures/whatsapp.png" alt="whatsapp" />
                <img src="/images/footer-pictures/telegram.png" alt="telegram" />
            </div>
        </div>
        </div>

        <div className={styles.contacts}>
            <h4 className={styles.title}>Контакты:</h4>
            <p>+7 (777) 490-00-91</p>
            <p>время работы: 9:00-20:00</p>
            <a href="">Заказать звонок</a>

            <div className={styles.mail}>
                <p>opt.sultan@mail.ru </p>
                <p>На связи в любое время</p>
            </div>

            <div className={styles.cards}>
                <img src="/images/footer-pictures/visa.png" alt="visa" />
                <img src="/images/footer-pictures/mastercard.png" alt="mastercard" />
            </div>
        </div>
      </div>
    </div>
  );
};

export default Footer;
