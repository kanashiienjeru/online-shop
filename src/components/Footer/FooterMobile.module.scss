// @type

.footer {
    padding: 15px;
    background: #3F4E65;
    
    .aboutUs {
        font-family: 'Inter';
        font-style: normal;
        color: #FFFFFF;

        .aboutUsHead {
            display: flex;
            align-items: center;
            justify-content: space-between;
            flex-wrap: wrap;
            margin-bottom: 15px;
            
            @media (max-width: 340px) {
                justify-content: center;
                gap: 15px;
            }

            button {
                width: 150px;
                height: 40px;

                background: #FFC85E;
                border-radius: 79px;

                font-weight: 700;
                font-size: 12px;
                line-height: 15px;
                
                display: flex;
                justify-content: center;
                align-items: center;
            }
        }

        p {
            font-weight: 400;
            font-size: 12px;
            line-height: 150%;
            margin-bottom: 15px;

            @media (min-width: 450px) {
                &:first-of-type {
                    text-align: center;
                }
            }
        }

        .input {
            position: relative;
            display: inline-block;
            width: 80%;
           
            
            input {
                width: inherit;
                box-sizing: content-box;
                margin: 0 auto;
                background-color: #fff;
                padding: 20px;
                padding-right: 25%;
                color: #000;
                border-radius: 36px;

                font-family: 'Inter';
                font-style: normal;
                font-weight: 400;
                font-size: 12px;

                @media (min-width: 420px) {
                    padding-right: 20%;
                }


                &::placeholder {
                    font-family: 'Inter';
                    font-style: normal;
                    font-weight: 400;
                    font-size: 12px;
                    line-height: 150%;
                    color: rgba(92, 99, 112, 0.6);

                }
            }

            img {
                position: absolute;
                top: 15%;
                right: -8%;
                padding: 10px 15px;
                background: #FFC85E;
                border-radius: 1000px;
                margin: 0;
                cursor: Pointer;

                @media (min-width: 420px) {
                    right: -2%;
                }

            }
        }

    }

    .links {
        margin-top: 30px;
        display: flex;
        justify-content: space-between;
        align-items: flex-start;
        gap: 24px;

        font-family: 'Inter';
        font-style: normal;
        color: #FFFFFF;


        .title {
            font-weight: 600;
            font-size: 20px;
            line-height: 150%;
            margin-bottom: 20px;
        }

        a {
            display: block;
            font-weight: 400;
            font-size: 12px;
            line-height: 150%;

            &:not(:last-of-type) {
                margin-bottom: 20px;
            }
        }
    }

    .contacts {
        font-family: 'Inter';
        font-style: normal;
        color: #FFFFFF;

        .title {
            font-weight: 600;
            font-size: 20px;
            line-height: 150%;
            margin-bottom: 25px;
        }
        .contactsHead {
            display: flex;
            justify-content: space-between;
            align-items: flex-start;
            gap: 27px;

            .call {
                margin-bottom: 25px;
                p:first-of-type {
                    font-weight: 600;
                    font-size: 14px;
                    line-height: 150%;
                    margin-bottom: 5px;
                }
                p:last-of-type {
                    font-weight: 300;
                    font-size: 12px;
                    line-height: 150%;
                    margin-bottom: 5px;
                    width: 152px;
                }
                a {
                    font-weight: 400;
                    font-size: 10px;
                    line-height: 150%;
                    text-decoration-line: underline;
                }
            }
            .socials {
                p {
                    font-weight: 400;
                    font-size: 14px;
                    line-height: 150%;
                    margin-bottom: 17px;
                }
                .socialsList {
                    display: flex;
                    align-items: center;
                    gap: 10px;
                }
            }
        }

        .mail {
            p:first-of-type {
                font-weight: 600;
                font-size: 14px;
                line-height: 18px;
            }
            P:last-of-type {
                font-weight: 300;
                font-size: 12px;
                line-height: 15px;
            }
        }

        .cards {
            margin-top: 10px;
            display: flex;
            align-items: center;
            gap: 5px;
        }
    }
}