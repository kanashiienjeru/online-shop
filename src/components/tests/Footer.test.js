import { render, screen } from "@testing-library/react";
import * as reduxHooks from '../../redux/hooks'
import Footer from "../Footer/Footer";

jest.mock('react-redux')
describe('Footer', () => {
    it('should render Footer', () => {
        const component = render(<Footer />)
        expect(component).toMatchSnapshot()
    })

    it('heading should have a white color', () => {
        const component = render(<Footer />)

        const headingList = screen.getAllByRole('heading')
        expect(component).toMatchSnapshot()
        headingList.forEach(heading => {
            expect(heading).toHaveClass('title')
        })
    })
})