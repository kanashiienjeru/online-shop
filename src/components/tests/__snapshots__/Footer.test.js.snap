// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer heading should have a white color 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <div
          class="content"
        >
          <div
            class="aboutUs"
          >
            <img
              alt="logo"
              src="/images/logo/whiteLogo.svg"
            />
            <p>
              Компания «Султан» — снабжаем розничные магазины товарами "под ключ" в Кокчетаве и Акмолинской области
            </p>
            <p>
              Подпишись на скидки и акции
            </p>
            <div
              class="input"
            >
              <input
                placeholder="Введите ваш E-mail"
                type="email"
              />
              <img
                alt="arrow"
                src="/images/arrows/arrow.svg"
              />
            </div>
          </div>
          <div
            class="menu"
          >
            <h4
              class="title"
              role="heading"
            >
              Меню сайта:
            </h4>
            <p>
              О компании
            </p>
            <p>
              Доставка и оплата
            </p>
            <p>
              Возврат
            </p>
            <p>
              Контакты
            </p>
          </div>
          <div
            class="categories"
          >
            <h4
              class="title"
              role="heading"
            >
              Категории:
            </h4>
            <p>
              Бытовая химия
            </p>
            <p>
              Косметика и гигиена
            </p>
            <p>
              Товары для дома
            </p>
            <p>
              Товары для детей и мам
            </p>
            <p>
              Посуда
            </p>
          </div>
          <div
            class="priceList"
          >
            <h4
              class="title"
              role="heading"
            >
              Скачать прайс-лист:
            </h4>
            <button>
              Прайс-лист
              <img
                alt="download"
                src="/images/icons/download.svg"
              />
            </button>
            <div
              class="socials"
            >
              <p>
                Связь в мессенджерах:
              </p>
              <div
                class="socialsList"
              >
                <img
                  alt="whatsapp"
                  src="/images/footer-pictures/whatsapp.png"
                />
                <img
                  alt="telegram"
                  src="/images/footer-pictures/telegram.png"
                />
              </div>
            </div>
          </div>
          <div
            class="contacts"
          >
            <h4
              class="title"
              role="heading"
            >
              Контакты:
            </h4>
            <p>
              +7 (777) 490-00-91
            </p>
            <p>
              время работы: 9:00-20:00
            </p>
            <a
              href=""
            >
              Заказать звонок
            </a>
            <div
              class="mail"
            >
              <p>
                opt.sultan@mail.ru 
              </p>
              <p>
                На связи в любое время
              </p>
            </div>
            <div
              class="cards"
            >
              <img
                alt="visa"
                src="/images/footer-pictures/visa.png"
              />
              <img
                alt="mastercard"
                src="/images/footer-pictures/mastercard.png"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <div
        class="content"
      >
        <div
          class="aboutUs"
        >
          <img
            alt="logo"
            src="/images/logo/whiteLogo.svg"
          />
          <p>
            Компания «Султан» — снабжаем розничные магазины товарами "под ключ" в Кокчетаве и Акмолинской области
          </p>
          <p>
            Подпишись на скидки и акции
          </p>
          <div
            class="input"
          >
            <input
              placeholder="Введите ваш E-mail"
              type="email"
            />
            <img
              alt="arrow"
              src="/images/arrows/arrow.svg"
            />
          </div>
        </div>
        <div
          class="menu"
        >
          <h4
            class="title"
            role="heading"
          >
            Меню сайта:
          </h4>
          <p>
            О компании
          </p>
          <p>
            Доставка и оплата
          </p>
          <p>
            Возврат
          </p>
          <p>
            Контакты
          </p>
        </div>
        <div
          class="categories"
        >
          <h4
            class="title"
            role="heading"
          >
            Категории:
          </h4>
          <p>
            Бытовая химия
          </p>
          <p>
            Косметика и гигиена
          </p>
          <p>
            Товары для дома
          </p>
          <p>
            Товары для детей и мам
          </p>
          <p>
            Посуда
          </p>
        </div>
        <div
          class="priceList"
        >
          <h4
            class="title"
            role="heading"
          >
            Скачать прайс-лист:
          </h4>
          <button>
            Прайс-лист
            <img
              alt="download"
              src="/images/icons/download.svg"
            />
          </button>
          <div
            class="socials"
          >
            <p>
              Связь в мессенджерах:
            </p>
            <div
              class="socialsList"
            >
              <img
                alt="whatsapp"
                src="/images/footer-pictures/whatsapp.png"
              />
              <img
                alt="telegram"
                src="/images/footer-pictures/telegram.png"
              />
            </div>
          </div>
        </div>
        <div
          class="contacts"
        >
          <h4
            class="title"
            role="heading"
          >
            Контакты:
          </h4>
          <p>
            +7 (777) 490-00-91
          </p>
          <p>
            время работы: 9:00-20:00
          </p>
          <a
            href=""
          >
            Заказать звонок
          </a>
          <div
            class="mail"
          >
            <p>
              opt.sultan@mail.ru 
            </p>
            <p>
              На связи в любое время
            </p>
          </div>
          <div
            class="cards"
          >
            <img
              alt="visa"
              src="/images/footer-pictures/visa.png"
            />
            <img
              alt="mastercard"
              src="/images/footer-pictures/mastercard.png"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Footer should render Footer 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <div
          class="content"
        >
          <div
            class="aboutUs"
          >
            <img
              alt="logo"
              src="/images/logo/whiteLogo.svg"
            />
            <p>
              Компания «Султан» — снабжаем розничные магазины товарами "под ключ" в Кокчетаве и Акмолинской области
            </p>
            <p>
              Подпишись на скидки и акции
            </p>
            <div
              class="input"
            >
              <input
                placeholder="Введите ваш E-mail"
                type="email"
              />
              <img
                alt="arrow"
                src="/images/arrows/arrow.svg"
              />
            </div>
          </div>
          <div
            class="menu"
          >
            <h4
              class="title"
              role="heading"
            >
              Меню сайта:
            </h4>
            <p>
              О компании
            </p>
            <p>
              Доставка и оплата
            </p>
            <p>
              Возврат
            </p>
            <p>
              Контакты
            </p>
          </div>
          <div
            class="categories"
          >
            <h4
              class="title"
              role="heading"
            >
              Категории:
            </h4>
            <p>
              Бытовая химия
            </p>
            <p>
              Косметика и гигиена
            </p>
            <p>
              Товары для дома
            </p>
            <p>
              Товары для детей и мам
            </p>
            <p>
              Посуда
            </p>
          </div>
          <div
            class="priceList"
          >
            <h4
              class="title"
              role="heading"
            >
              Скачать прайс-лист:
            </h4>
            <button>
              Прайс-лист
              <img
                alt="download"
                src="/images/icons/download.svg"
              />
            </button>
            <div
              class="socials"
            >
              <p>
                Связь в мессенджерах:
              </p>
              <div
                class="socialsList"
              >
                <img
                  alt="whatsapp"
                  src="/images/footer-pictures/whatsapp.png"
                />
                <img
                  alt="telegram"
                  src="/images/footer-pictures/telegram.png"
                />
              </div>
            </div>
          </div>
          <div
            class="contacts"
          >
            <h4
              class="title"
              role="heading"
            >
              Контакты:
            </h4>
            <p>
              +7 (777) 490-00-91
            </p>
            <p>
              время работы: 9:00-20:00
            </p>
            <a
              href=""
            >
              Заказать звонок
            </a>
            <div
              class="mail"
            >
              <p>
                opt.sultan@mail.ru 
              </p>
              <p>
                На связи в любое время
              </p>
            </div>
            <div
              class="cards"
            >
              <img
                alt="visa"
                src="/images/footer-pictures/visa.png"
              />
              <img
                alt="mastercard"
                src="/images/footer-pictures/mastercard.png"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <div
        class="content"
      >
        <div
          class="aboutUs"
        >
          <img
            alt="logo"
            src="/images/logo/whiteLogo.svg"
          />
          <p>
            Компания «Султан» — снабжаем розничные магазины товарами "под ключ" в Кокчетаве и Акмолинской области
          </p>
          <p>
            Подпишись на скидки и акции
          </p>
          <div
            class="input"
          >
            <input
              placeholder="Введите ваш E-mail"
              type="email"
            />
            <img
              alt="arrow"
              src="/images/arrows/arrow.svg"
            />
          </div>
        </div>
        <div
          class="menu"
        >
          <h4
            class="title"
            role="heading"
          >
            Меню сайта:
          </h4>
          <p>
            О компании
          </p>
          <p>
            Доставка и оплата
          </p>
          <p>
            Возврат
          </p>
          <p>
            Контакты
          </p>
        </div>
        <div
          class="categories"
        >
          <h4
            class="title"
            role="heading"
          >
            Категории:
          </h4>
          <p>
            Бытовая химия
          </p>
          <p>
            Косметика и гигиена
          </p>
          <p>
            Товары для дома
          </p>
          <p>
            Товары для детей и мам
          </p>
          <p>
            Посуда
          </p>
        </div>
        <div
          class="priceList"
        >
          <h4
            class="title"
            role="heading"
          >
            Скачать прайс-лист:
          </h4>
          <button>
            Прайс-лист
            <img
              alt="download"
              src="/images/icons/download.svg"
            />
          </button>
          <div
            class="socials"
          >
            <p>
              Связь в мессенджерах:
            </p>
            <div
              class="socialsList"
            >
              <img
                alt="whatsapp"
                src="/images/footer-pictures/whatsapp.png"
              />
              <img
                alt="telegram"
                src="/images/footer-pictures/telegram.png"
              />
            </div>
          </div>
        </div>
        <div
          class="contacts"
        >
          <h4
            class="title"
            role="heading"
          >
            Контакты:
          </h4>
          <p>
            +7 (777) 490-00-91
          </p>
          <p>
            время работы: 9:00-20:00
          </p>
          <a
            href=""
          >
            Заказать звонок
          </a>
          <div
            class="mail"
          >
            <p>
              opt.sultan@mail.ru 
            </p>
            <p>
              На связи в любое время
            </p>
          </div>
          <div
            class="cards"
          >
            <img
              alt="visa"
              src="/images/footer-pictures/visa.png"
            />
            <img
              alt="mastercard"
              src="/images/footer-pictures/mastercard.png"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
